web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -377,18 +377,41 @@ const isOwner = computed(() => !!(
 ));
 const itemsNotFound = computed(() => items.value && !items.value.length);
 
+function replaceStringsInEndpoint(
+  endpoint: string,
+  substitutions: {[key: string]: string} | undefined,
+) {
+  if (!substitutions) return endpoint;
+  let result = endpoint;
+  Object.entries(substitutions).forEach(([key, value]) => {
+    result = result.replace(key, value);
+  });
+  return result;
+}
+
+const isStaging = process.env.VUE_APP_SENTRY_ENVIRONMENT === 'staging';
+
 function getExternalServices(path: AssetPath) {
   const servicePredicate = (service: Service, _path: AssetPath) => (
     new RegExp(service.regex).test(path.path)
           && _path.asset !== null
           && _path.aggregate_size <= service.maxsize
   );
 
+  const baseApiUrl = isStaging
+    ? 'https://api-staging.dandiarchive.org'
+    : 'https://api.dandiarchive.org';
```

Comment:
Let's avoid harcoding any url:

```suggestion

function getExternalServices(path: AssetPath) {
  const servicePredicate = (service: Service, _path: AssetPath) => (
    new RegExp(service.regex).test(path.path)
          && _path.asset !== null
          && _path.aggregate_size <= service.maxsize
  );

  const baseApiUrl = process.env.VUE_APP_DANDI_API_ROOT;
```
and adjust below according to suggestion?